src/applications/ask-va/components/MedicalFacilitySearch.jsx

```diff
@@ -5,9 +5,125 @@ import { convertToLatLng } from '../utils/mapbox';
+//           saturday: 'Closed',
+//           sunday: 'Closed',
+//         },
+//         operatingStatus: {
+//           code: 'NORMAL',
+//         },
+//         phone: {
+//           fax: '970-962-4901',
+//           main: '970-224-1550',
+//           pharmacy: '866-420-6337',
+//           afterHours: '307-778-7550',
+//           patientAdvocate: '307-778-7550 x7573',
+//           mentalHealthClinic: '307-778-7349',
+//           enrollmentCoordinator: '970-593-3311',
+//         },
+//         services: {
+//           health: [
+//             {
+//               name: 'My HealtheVet coordinator',
+//               serviceId: 'myHealtheVetCoordinator',
+//               link:
+//                 'https://sandbox-api.va.gov/services/va_facilities/v1/facilities/vha_442GC/services/myHealtheVetCoordinator',
+//             },
+//             {
+//               name: 'Nutrition, food, and dietary care',
+//               serviceId: 'nutrition',
+//               link:
+//                 'https://sandbox-api.va.gov/services/va_facilities/v1/facilities/vha_442GC/services/nutrition',
+//             },
+//             {
+//               name: 'Pharmacy',
+//               serviceId: 'pharmacy',
+//               link:
+//                 'https://sandbox-api.va.gov/services/va_facilities/v1/facilities/vha_442GC/services/pharmacy',
+//             },
+//             {
+//               name: 'Primary care',
+//               serviceId: 'primaryCare',
+//               link:
+//                 'https://sandbox-api.va.gov/services/va_facilities/v1/facilities/vha_442GC/services/primaryCare',
+//             },
+//             {
+//               name: 'Social work',
+//               serviceId: 'socialWork',
+//               link:
+//                 'https://sandbox-api.va.gov/services/va_facilities/v1/facilities/vha_442GC/services/socialWork',
+//             },
+//           ],
+//           link:
+//             'https://sandbox-api.va.gov/services/va_facilities/v1/facilities/vha_442GC/services',
+//           lastUpdated: '2024-07-24',
+//         },
+//       },
+//     },
+//   ],
+//   meta: {
+//     pagination: {
+//       current_page: 1,
+//       prev_page: null,
+//       next_page: 2,
+//       total_pages: 139,
+//       total_entries: 1381,
+//     },
+//   },
+//   links: {
+//     self:
+//       'https://dev-api.va.gov/ask_va_api/v0/health_facilities?lat=40.553874&long=-105.087951&page=1&per_page=10&state=CO&type=health',
+//     first:
+//       'https://dev-api.va.gov/ask_va_api/v0/health_facilities?lat=40.553874&long=-105.087951&per_page=10&state=CO&type=health',
+//     prev: null,
+//     next:
+//       'https://dev-api.va.gov/ask_va_api/v0/health_facilities?lat=40.553874&long=-105.087951&page=2&per_page=10&state=CO&type=health',
+//     last:
+//       'https://dev-api.va.gov/ask_va_api/v0/health_facilities?lat=40.553874&long=-105.087951&page=139&per_page=10&state=CO&type=health',
+//   },
+// };
```

Comment:
Is this meant to serve as a reference for how data is structured? If so, consider keeping in `/fixtures/data` folder similar to [this example](https://github.com/department-of-veterans-affairs/vets-website/blob/5340e2608d43879ca1c897dcdd59f36a7776df03/src/applications/appeals/996/tests/fixtures/data/maximal-test-v1.json#L2)

Reply:
Was for the data structure temporarily, removed